dev-libs/libassuan/libassuan-2.5.1.recipe

```diff
@@ -1,26 +1,28 @@
 SUMMARY="IPC library used by GnuPG and GPGME"
-DESCRIPTION="
-Libassuan is a small library implementing the so-called Assuan protocol. \
-This protocol is used for IPC between most newer GnuPG components. \
-Both, server and client side functions are provided.
-"
-COPYRIGHT="2003-2013 g10 Code GmbH"
+DESCRIPTION="Libassuan is a small library implementing the so-called \
+Assuan protocol. This protocol is used for IPC between most newer GnuPG \
+components. Both, server and client side functions are provided."
+COPYRIGHT="2001-2013 Free Software Foundation, Inc.
+	2001-2015 g10 Code GmbH"
 LICENSE="
 	GNU GPL v3
 	GNU LGPL v2.1
 	"
 REVISION="1"
-HOMEPAGE="http://www.gnupg.org/related_software/libassuan/index.en.html"
+HOMEPAGE="https://www.gnupg.org/related_software/libassuan/index.en.html"
```

Comment:
```diff
-HOMEPAGE="https://www.gnupg.org/related_software/libassuan/index.en.html"
+HOMEPAGE="https://www.gnupg.org/related_software/libassuan/"
```
This is better for 3 reasons:
* The webmaster already has `index.en.html` as default page and sends it without forcing a redirect.
* Should the webmaster add a translated page for another language spoken in the country where the client is, he would receive the content for the translated page (or be redirected to the alternate page).
* This allows the webmaster to switch from HTML to PHP (or any other language), should such a decision be taken.

So for non english-speaking users, it is better to not specify any language in `HOMEPAGE`.